run_tests

```diff
@@ -3,7 +3,7 @@ set -e # If anything fails the script fails
 
 
 export SECRET_KEY='superfakesecretkey'
-export DJANGO_SETTINGS_MODULE='course_gather.settings.prod_settings'
+export DJANGO_SETTINGS_MODULE='course_gather.settings.dev_settings'
```

Comment:
Why change the tests configuration at all?

Reply:
Otherwise, we'd need to set up a Postgres database for the tests.